Automattic/woocommerce-payments
# Update references to fraudulent disputes

Fixes #6611

## Changes proposed in this Pull Request

We recently retitled "fraudulent" disputes to "unauthorized transactions" in https://github.com/Automattic/woocommerce-payments/issues/6372. There are some additional references to fraudulent disputes which need to be updated.



### Changes made

#### Fraudulent disputes

![image](https://github.com/Automattic/woocommerce-payments/assets/57298/56588bfa-1c2c-44cf-a1ac-5a4b203cf595)

**The overview section**
```diff
-If you believe the payment was actually made using a stolen credit card, you will need to accept the dispute. The credit card networks place liability for accepting fraudulent payments with you, the business.
+If you believe the payment was actually made using a stolen credit card, you will need to accept the dispute. The credit card networks place liability for accepting unauthorized transactions with you, the business.
```

**Summary**
```diff
-This is the most common reason for a dispute, and happens when a cardholder claims that they didn’t authorize the payment. This can happen if the card was lost or stolen and used to make a fraudulent purchase. It can also happen if the cardholder doesn’t recognize the payment as it appears on the billing statement from their card issuer.
+This is the most common reason for a dispute, and happens when a cardholder claims that they didn’t authorize the payment. This can happen if the card was lost or stolen and used to make an unauthorized transaction. It can also happen if the cardholder doesn’t recognize the payment as it appears on the billing statement from their card issuer.
```

#### Unrecognized disputes

![image](https://github.com/Automattic/woocommerce-payments/assets/57298/09c9cc8d-01b3-4f8a-8e41-af3d1125c60a)
This is a mockup based on the text in WCPay and is not currently testable.

**Requirements** This section was likely pulled from a document describing how to handle different dispute types. In place, it makes no sense as the reference to fraudulent disputes is not visible. 
```diff
-As with fraudulent disputes, get your customer to withdraw the dispute by helping them identify the payment. Otherwise challenge the dispute with appropriate evidence that proves the purchase was legitimate.
+Get your customer to withdraw the dispute by helping them identify the payment. Otherwise, challenge the dispute with appropriate evidence that proves the purchase was legitimate.
```

#### Set up refund policy - Note

![image](https://github.com/Automattic/woocommerce-payments/assets/57298/bd0eb9fc-175d-4995-b4a6-213887732829)


```diff
-Protect your merchant account from fraudulent disputes by defining the policy and making it accessible to customers.
+Protect your merchant account from unauthorized transaction disputes by defining the policy and making it accessible to customers.
```

### Testing instructions

1. Purchase a product using a ["dispute" test card](https://stripe.com/docs/testing#disputes) `4000000000000259` will automatically create a dispute with a reason of `fraudulent`
2. View the dispute under Payments > Disputes, and click through to dispute details.
3. There should be no references to `fraudulent`

To test the refund policy note:
Create a new store, or run the SQL: `DELETE FROM wp_wc_admin_notes WHERE name = 'wc-payments-notes-set-up-refund-policy';`
The note can then be seen under WC Home


-------------------


**Post merge**
